src/d_loop.c

```diff
@@ -817,7 +819,7 @@ void TryRunTics (void)
             // forever - give the menu a chance to work.
             if (I_GetTime() / ticdup - entertic >= MAX_NETGAME_STALL_TICS)
             {
-                return;
+                goto end;
```

Comment:
We don't call `RunTic()` here and in the case of `!PlayersInGame()`, so no sounds will be updated. So I guess we don't need `goto`, right?

Reply:
Maybe I'm being too cautious. Should both `goto` lines be changed back to `return`?

When `S_UpdateSounds()` was originally outside in the `D_DoomMain()` loop, it would still be called when returning from `TryRunTics()` at this point, even with a capped fps.

Reply:
A few lines up `NetUpdate()` is called every loop iteration until this timeout occurs. `NetUpdate` calls `BuildNewTic()`. This is difficult to follow.

Reply:
> Should both `goto` lines be changed back to `return`?

Yes, I think it's safe to change it back to `return`.

> `NetUpdate` calls `BuildNewTic()`

`BuildNewTic()` does not call the `*_Ticker()` functions, where `S_StartSound()` is called.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1118/commits/06f271ccd7dfa829a4bd54c1dc41e0b178e2b889